vmm_tests/vmm_tests/tests/tests/x86_64/openhcl_linux_direct.rs

```diff
@@ -385,26 +385,58 @@ async fn storvsp(config: PetriVmBuilder<OpenVmmPetriBackend>) -> Result<(), anyh
 
     let sh = agent.unix_shell();
     // The drive ordering is not guaranteed, so we need to check all drives.
-    let output = cmd!(sh, "sh -c 'cat /sys/block/sd*/size'").read().await?;
+    let cmd = r#"
+exec 2>&1;
+for device in /sys/block/sd*; do
```

Comment:
This is fine if that's how you want to do it. But I wonder if we should make it easier to do this via Rust style code. E.g., would you prefer to write code like this if you could?

```rust
for device in sh.glob("/sys/block/sd*").await? {
    let size = sh.read(device.join("size")).await?.parse::<u64>().context("whatever")?;
    etc...
}
```